hostinfo/fileperms.lua

```diff
@@ -0,0 +1,84 @@
+--[[
+Copyright 2015 Rackspace
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+]]--
+local table = require('table')
+local los = require('los')
+local async = require('async')
+local fs = require('fs')
+local execFileToBuffers = require('./misc').execFileToBuffers
+
+--[[ Check file permissions ]]--
+local HostInfo = require('./base').HostInfo
+local Info = HostInfo:extend()
+function Info:initialize()
+    HostInfo.initialize(self)
+end
+
+function Info:run(callback)
+    if los.type() ~= 'linux' then
```

Comment:
Wouldn't we want to support all unix flavors that tend to have these files?  I'm sure mosts unix hosts are linux, but there may be some BSD, Darwin, or Solaris variants.